packages/react/src/Card/Card.tsx

```diff
@@ -0,0 +1,95 @@
+import React, {forwardRef} from 'react'
+import {isFragment} from 'react-is'
+import clsx from 'clsx'
+
+import {Heading, Text, Link} from '..'
+import type {BaseProps} from '../component-helpers'
+
+/**
+ * Design tokens
+ */
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/card/base.css'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/card/colors-with-modes.css'
+
+/**
+ * Main stylesheet (as a CSS Module)
+ */
+import styles from './Card.module.css'
+
+export type CardProps = {
+  /**
+   * Valid children include Card.Image, Card.Heading, and Card.Description
+   */
+  children: React.ReactNode | React.ReactElement<CardHeadingProps> | React.ReactElement<CardDescriptionProps>
+  /**
+   * Changes the link content of the card
+   * * @default 'Learn more'
+   * */
+  link?: string
+  /**
+   * The href of the link
+   * */
+  linkHref: string
+} & BaseProps<HTMLLinkElement>
+
+const CardRoot = forwardRef<HTMLLinkElement, CardProps>(({children, className, link, linkHref}) => {
+  const childrenHasFragment = React.Children.toArray(children).some(child => isFragment(child))
+  const filteredChildren = React.Children.toArray(children).filter(child => {
+    if (React.isValidElement(child) && typeof child.type !== 'string') {
+      if (childrenHasFragment || child.type === CardHeading || child.type === CardDescription) {
+        return true
+      }
+    }
+    return false
+  })
+
+  return (
+    <a href={linkHref} className={clsx(styles.Card, className)}>
```

Comment:
Could we forward all native `a` element attributes here too using spread? Imagine things like `target` will become useful in some use-cases, as well as `referrer`.

Reply:
Done!